src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -972,16 +972,29 @@ def customizePerJob(self, job):
 
         # Performance estimates
         if job.get('estimatedJobTime'):
-            jdl.append('+MaxWallTimeMins = %d\n' % int(job['estimatedJobTime'] / 60.0))
+            estimatedMins = int(job['estimatedJobTime'] / 60.0)
+            origCores = int(job.get('numberOfCores', 1))
+            estimatedMinsSingleCore = estimatedMins*origCores
+            jdl.append('+MaxWallTimeMins = WMCore_ResizeJob ? (%d/RequestCpus + 60) : %d\n' % (estimatedMinsSingleCore, estimatedMins))
+            
         if job.get('estimatedMemoryUsage'):
-            jdl.append('request_memory = %d\n' % int(job['estimatedMemoryUsage']))
+            extraMem = int(job.get('extraMemoryPerCore', 500))
+            baseMem = int(job['estimatedMemoryUsage'])
+            jdl.append('+RequestMemory = WMCore_ResizeJob ? (%d + RequestCpus*%d) : %d\n' % (baseMem, extraMem, baseMem))
```

Comment:
As I said during your presentation, this should be

jdl.append('+RequestMemory = WMCore_ResizeJob ? (%d + (RequestCpus-1)*%d) : %d\n' % (baseMem, extraMem, baseMem))

since baseMem will already include the relevant extraMem portion for a singlecore job.